stream-chat-android-client/build.gradle

```diff
@@ -56,6 +56,8 @@ tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).all {
         freeCompilerArgs += [
                 '-progressive',
                 '-Xexplicit-api=strict',
+                '-Xopt-in=kotlin.RequiresOptIn',
+                '-Xopt-in=io.getstream.chat.android.client.internal.InternalStreamChatApi',
```

Comment:
Adding the annotation as a compiler argument enables it for the entire module, so there's no opt-in required for each statement/class/file that uses it